### ISSUE TITLE
Undefined op in speech_commands example

### ISSUE THREAD

**Issue by ydzhang12345** [tensorflow/tensorflow#12657]
I am using tensorflow 1.3.0 built from source.

I run the newly released speech_commands example (its in tensorflow/examples/speech_commands), following all the stuffs and all good. However, after I used the freeze.py script and generated the pb graph, I am unable to load it back. The code I used to load pb graph is:

```
def load_graph(frozen_graph_filename):
    with tf.gfile.GFile(frozen_graph_filename, "rb") as f:
        graph_def = tf.GraphDef()
        graph_def.ParseFromString(f.read())

    with tf.Graph().as_default() as graph:
        tf.import_graph_def(
            graph_def, 
            input_map=None, 
            return_elements=None, 
            name="imported", 
            op_dict=None, 
            producer_op_list=None
        )
    return graph
graph = load_graph('./test_loading_graph_back.pb')
```

Error:
File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/importer.py", line 285, in import_graph_def
    raise ValueError('No op named %s in defined operations.' % node.op)
ValueError: No op named DecodeWav in defined operations.

**Comment by jart** [comment]
Please provide details about what platform you are using  (operating system, architecture). Also include your TensorFlow version. Also, did you compile from source or install a binary?  Make sure you also include the exact command if possible to produce  the output included in your test case. If you are unclear what to include  see the issue template displayed in  [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new).

 We ask for this in the issue submission template, because    it is really difficult to help without that information. Thanks!

**Comment by ydzhang12345** [comment]
-  I haven't written customized code
- **OS Platform and Distribution: Linux Ubuntu 14.04
- **TensorFlow installed from source
- **TensorFlow version 1.3.0
- **Python version: 2.7 
- **Bazel version (if compiling from source)**: 0.5.3
- **CUDA:8.0 cuDNN:6.0**:
- **GPU: GTX1080**:
- **Exact command to reproduce**:

1. built tensorflow from source

2. run the tensorflow/examples/speech_commands with the default setting to train CNN model;

3. use the freeze.py script inside the speech_commands dir to obtain frozen_graph.pb

4. Load the graph back by using the following code:
`def load_graph(frozen_graph_filename):
    with tf.gfile.GFile(frozen_graph_filename, "rb") as f:
        graph_def = tf.GraphDef()
        graph_def.ParseFromString(f.read())

    with tf.Graph().as_default() as graph:
        tf.import_graph_def(
            graph_def, 
            input_map=None, 
            return_elements=None, 
            name="imported", 
            op_dict=None, 
            producer_op_list=None
        )
    return graph
graph = load_graph('./test_loading_graph_back.pb')`

Error:
File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/importer.py", line 285, in import_graph_def
raise ValueError('No op named %s in defined operations.' % node.op)
ValueError: No op named DecodeWav in defined operations.

**Comment by jart** [comment]
Gosh is there any chance you're doing something related to mobile? Like Android? Also is it possible that this graph was saved using a different version of TensorFlow? It's just strange because the DecodeWav op is in core.

**Comment by keryil** [comment]
I am running into the same issue on 1.4 rc (compiled from source, bazel v0.7.0). At first I thought it was trying to load the model on a Raspberry Pi that caused this, but it looks like I cannot load the frozen model even on the very workstation it was created on. Any tips on how to deal with this?

**Comment by keryil** [comment]
Sorry for the late reply. I have tried again with a freshly installed TF 1.4, and whatever my problem was, it seems to exist no more. Just a heads up if anybody encounters something similar.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by ydzhang12345** [comment]
it is resolved in TF 1.4. Thank you!

**Comment by beala** [comment]
I had the same issue on TF 1.4 with the speech_commands example. It turns out that I had absentmindedly removed this from the top of `label_wav.py`, thinking that it was unused:

```
# pylint: disable=unused-import
from tensorflow.contrib.framework.python.ops import audio_ops as contrib_audio
# pylint: enable=unused-import
```

It turns out having `contrib_audio.decode_wav` in scope is essential to loading the graph.

I realize this isn't exactly the same as the original issue, but perhaps others who are modifying `label_wav.py` or writing a custom loader will, as I did, find this through google.
